pages.tr/linux/xterm.md

```diff
@@ -0,0 +1,23 @@
+# xterm
+
+> X Ekran Sistemi için terminal öykünücüsü.
+
+- `Örnek` başlığına sahip bir terminal aç:
+
+`xterm -T {{Örnek}}`
+
+- Terminali tam ekran modunda aç:
+
+`xterm -fullscreen`
+
+- Terminali lacivert arkaplan ve sarı ön plan (font rengi) ile aç:
+
+`xterm -bg {{darkblue}} -fg {{yellow}}`
+
+- Terminali satır başına 100 karakter ve sütun başına 35 satır sığacak şekilde, x=200px y=20px koordinatlarında aç:
+
+`xterm -geometry {{100}}x{{35}}+{{200}}+{{20}}`
+
+- Terminali bir Serif fontu ve 20'ye eşit olan bir font büyüklüğü ile aç:
+
+`xterm -fa {{'Serif'}} -fs {{20}}`
```

Comment:
We like to put quotes ouside of the token syntax, and use double quotes when possible.  This might need to be changed in the original and all translations...
```suggestion
`xterm -fa "{{Serif}}" -fs {{20}}`
```